src/main/webapp/ratingScript.js

```diff
@@ -110,6 +112,34 @@ async function getLatestRating() {
   }
 }
 
+function validSubmission() {
```

Comment:
Nit: Usually functions returning a boolean like this have names like "isValidSubmission"